index.js

```diff
@@ -0,0 +1,146 @@
+const data = [
+  {
+    title: "Inception",
+    year: 2010,
+    rating: 8.8,
+    genre: ["Action", "Sci-Fi"],
+    director: "Christopher Nolan",
+  },
+  {
+    title: "The Dark Knight",
+    year: 2008,
+    rating: 9.0,
+    genre: ["Action", "Crime", "Drama"],
+    director: "Christopher Nolan",
+  },
+  {
+    title: "Pulp Fiction",
+    year: 1994,
+    rating: 8.9,
+    genre: ["Crime", "Drama"],
+    director: "Quentin Tarantino",
+  },
+  {
+    title: "The Matrix",
+    year: 1999,
+    rating: 8.7,
+    genre: ["Action", "Sci-Fi"],
+    director: "Lana Wachowski, Lilly Wachowski",
+  },
+  {
+    title: "Fight Club",
+    year: 1999,
+    rating: 8.8,
+    genre: ["Drama"],
+    director: "David Fincher",
+  },
+];
+
+//1 - Create an array of movie titles.
+const movieTitles = data.map((movie) => movie.title);
+console.log(movieTitles);
+
+//2 - create an array of movies with a rating higher than 8.5
+const moviesHigherThan8_5 = data.filter((movie) => movie.rating > 8.5);
+console.log(moviesHigherThan8_5);
+
+//3 - Sort the movies by rating in decending order.
+const sortedMovies = data.sort((a, b) => b.rating - a.rating);
+console.log(sortedMovies);
+
+//4 - Create an array of movies released after the year 2000.
+const moviesAfter2000 = data.filter((movie) => movie.year >= 2000);
```

Comment:
after the year 2000 means just greater than. it didnt say the year of and after.